### ISSUE TITLE
[BUG] Unable to start trigger using postman!

### ISSUE THREAD

**Issue by Prameelagr8** [elsa-workflows/elsa-core#5873]
## Description
Unable to start trigger using postman!

## Steps to Reproduce
In Elsa designer I have added a trigger, which looks like this.

![image](https://github.com/user-attachments/assets/279d4cb2-e49e-4da8-8293-b805d7686e87)

And started work flow and result is suspended:
![image](https://github.com/user-attachments/assets/1cbb1f66-549a-4158-aa2b-99247cfca6d2)

And now executing trigger using postman so that workflow completes:
![image](https://github.com/user-attachments/assets/346eec29-f75d-4c9f-97ba-9ec4ad44b586)

Here if we observer,respond body saying "an exception with link to reload". Please let me know if anything else need to be done to do postman request.

After this, I went back and verified but workflow is stil suspended which means trigger didnt execute using above request!

**Comment by sfmskywalker** [comment]
What happens here is that you started the workflow manually, which causes the Event activity to execute, which will create a Bookmark and suspends the workflow. To resume the workflow, you need to resume the bookmark and not trigger the workflow.

I recorded a small video that shows how it works: https://youtu.be/m2e3hjS2Nxw

**Comment by sfmskywalker** [comment]
Here's the Postman request I used:

```
curl --location 'https://localhost:5001/elsa/api/events/MyEvent/trigger' \
--header 'Content-Type: application/json' \
--header 'Authorization: ApiKey 48587230567A646D394B435A6277734A-4802fa49-e91e-45e8-b00f-b5492377e20b' \
--data '{
    
}'
```

And here's an export of the workflow:

[workflow-1.zip](https://github.com/user-attachments/files/16559550/workflow-1.zip)

**Comment by Prameelagr8** [comment]
I have imported your workflow and executed using postman after starting workflow from elsa studio. Stil same issue.
Here is curl response for your reference:
![image](https://github.com/user-attachments/assets/786b23c2-63d1-4448-9050-fd3fb2e30939)

**Comment by sfmskywalker** [comment]
It looks like you have a different project setup. The API request you made responds with HTML from the Elsa Studio project.

**Comment by Prameelagr8** [comment]
This is the project I am running to start both server and studio both.

![image](https://github.com/user-attachments/assets/05efb8f7-bed4-4961-bfcc-e04dd60e7c63)

**Comment by Prameelagr8** [comment]
Any help on this. My entire workflow is depending on this usecase!

**Comment by sfmskywalker** [comment]
I have tried that, too. Works fine. Are you on the `main` branch, or `patch/3.2.x`?

**Comment by Prameelagr8** [comment]
I am using Main branch

**Comment by sfmskywalker** [comment]
Yeah same here. If there's anything you can share in terms of steps to reproduce (from scratch) and or a sample project that you can zip and send over, I'd be happy to take a look.

**Comment by Prameelagr8** [comment]
I saw that if end point is incorrect we generally get above error i stated. Please let me know the actual end point to trigger "event". As of now I tried using following urls:
https://localhost:5001/api/testevent/trigger (POST -postman)
https://localhost:5001/elsa/api/testevent/trigger (POST - postman)
Tried with and with out giving body parameters.

Please correct me if anything need to be changed.
Parallelly, I tried same functionality using http but i need inputs. Please check Bug #5879 also.
